test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -202,13 +324,10 @@ public function invalidVersionStringsProvider() : array
     {
         return [
             [''],
-            ['12.a'],
-            ['12a3'],
             ['alpha'],
             ['beta'],
-            ['1-a'],
-            ['1.2.a'],
-            ['.1'],
+//            ['1.2.3-alpha.beta'], // it is actually a legit version, we just do not handle it for now
```

Comment:
Wait... what?! O.o

Reply:
yes it is - https://semver.org/#spec-item-11